test/parallel/test-fs-write-stream.js

```diff
@@ -52,16 +52,3 @@ tmpdir.refresh();
   });
   stream.destroy();
 }
-
-// Throws if data is not of type Buffer.
-{
-  const stream = fs.createWriteStream(file);
-  common.expectsError(() => {
-    stream._write(42, null, function() {});
-  }, {
-    code: 'ERR_INVALID_ARG_TYPE',
-    type: TypeError,
-    message: 'The "data" argument must be of type Buffer. Received type number'
-  });
-  stream.destroy();
-}
```

Comment:
Can you add a simple test where we check the error code in case of an invalid type?

Reply:
FIxed